docs/userGuide/syntax/scrollTopButton.md

```diff
@@ -0,0 +1,39 @@
+{% from "userGuide/components/advanced.md" import slot_info_trigger %}
+
+## Scroll To Top Button
+
+<div id="content">
+
+The ```scroll-top-button``` component allows users to move to the top of the page.
+
+****Adding a scroll-top-button****
+
+Add ```<scroll-top-button><scroll-top-button/>``` to layout file.
+
+****Options****
+
+| Name     | Type     | Default                 | Description                                                                           |
+|----------|----------|-------------------------|---------------------------------------------------------------------------------------|
+| icon     | `String` | `fa fa-arrow-circle-up` | Icon used for button.                                                                 |
+| iconSize | `String` | ``                      | Size of button. Supports integer-scaling of the icon dimensions e.g. 2x, 3x, 4x, etc. |
```

Comment:
Could we use kebab-case (`icon-size`) instead of camelCase here? camelCase is a bit of an antipattern in HTML attributes and may cause unexpected behaviour, such as with jQuery, because all attributes will get converted to lowercase ([HTML spec](https://html.spec.whatwg.org/multipage/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes)).
Also, we can have consistency with our other components like `<box icon-size="..."></box>

Reply:
Sorry @jovyntls did the comment get cut off?

Reply:
It did, sorry for the confusion 😅 Meant to say: other components like `Box`

Reply:
I left the default iconSize to be 'lg' instead of null, otherwise it will be too small. Is that ok?

Reply:
Yup that sounds good! Thanks @yucheng11122017  :)